src/main/scala/is/hail/methods/Nirvana.scala

```diff
@@ -337,22 +339,18 @@ object Nirvana {
         it.map { case (v, nirvana) =>
           rvb.start(nirvanaRowType)
           rvb.startStruct()
-          rvb.addAnnotation(nirvanaRowType.fieldType(0), pkProjection(v))
-          rvb.addAnnotation(nirvanaRowType.fieldType(1), v)
+          rvb.startStruct()
+          rvb.addAnnotation(nirvanaRowType.fieldType(0), v.locus)
+          rvb.addAnnotation(nirvanaRowType.fieldType(1), IndexedSeq(v.ref) ++ v.altAlleles.map(_.alt))
           rvb.addAnnotation(nirvanaRowType.fieldType(2), nirvana)
           rvb.endStruct()
           rv.setOffset(rvb.end())
 
           rv
         }})
 
-    val (newVAType, inserter) = vds.vaSignature.structInsert(nirvanaSignature, parsedRoot)
-
-    val newMatrixType = vds.matrixType.copy(vaType = newVAType)
-    val newRDD2 = vds.orderedRVDLeftJoinDistinctAndInsert(vds.rdd2, nirvanaRVD,
-      newMatrixType.orderedRVType, product = false, 2, newVAType, inserter)
-
-    vds.copy2(rdd2 = newRDD2, vaSignature = newVAType)
+    vds.orderedRVDLeftJoinDistinctAndInsert(nirvanaRVD, "nirvana", product = false)
+      .annotateVariantsExpr("vep = va.vep.vep")
```

Comment:
Ah, this can't be right, we're in nirvana.

Reply:
I tested VEP before the CGA demo, just to note.